generator/Dockerfile

```diff
@@ -1,17 +1,19 @@
 FROM golang:latest
 
-RUN apt-get update && apt-get install -y libsnmp-dev
-
-RUN go get github.com/prometheus/snmp_exporter/generator
-RUN cd /go/src/github.com/prometheus/snmp_exporter/generator \
+RUN apt-get update && \
+    apt-get install -y libsnmp-dev && \
+    go get github.com/prometheus/snmp_exporter/generator && \
+    cd /go/src/github.com/prometheus/snmp_exporter/generator && \
     go get -v . && \
-    go install
-
-# Move the default generator file to the expected location
-RUN cp /go/src/github.com/prometheus/snmp_exporter/generator/generator.yml /opt/
+    go install && \
+    cp generator.yml /opt/ && \
+    make mibs && \
+    mv mibs /opt/
 
 WORKDIR "/opt"
 
 ENTRYPOINT ["/go/bin/generator"]
 
+ENV MIBDIRS mibs
```

Comment:
Actually, this values doesn't match the docs. This should work off the user's mibs, not the ones we use.

Reply:
I've update the README to make things easier and match our build.